Source/driver/Castro_io.cpp

```diff
@@ -878,9 +878,21 @@ Castro::writeJobInfo (const std::string& dir)
   jobInfoFile << " Inputs File Parameters\n";
   jobInfoFile << PrettyLine;
 
-  ParmParse::dumpTable(jobInfoFile, true);
+  //ParmParse::dumpTable(jobInfoFile, true);
```

Comment:
What's going on here?

Reply:
oh, I can remove that.  That was the old way of outputting the params to job_info, but it only worked for those that were set in inputs.  This works now for everything that was defined in _cpp_parameters